packages/core/components/EditorPanel/VizFilterEditor/VizFilterEditor.tsx

```diff
@@ -80,7 +81,6 @@ const VizFilterEditor: React.FC<VizFilterProps> = ({ config, updateField, rawDat
     const filterItem = { ...filtersCopy[filterIndex] }
 
     // Overwrite filterItem.values since thats what we map through in the editor panel
-    filterItem.values = updatedValues

```

Comment:
@Matt-DiPaolo why as values removed in the first place? the comment above it should be updated if it needs to be removed. 

just not sure why we are making any changes to the VizFilterEditor unless there was an actual bug you found